app/services/HttpIdentityService.scala

```diff
@@ -204,7 +204,7 @@ class HttpIdentityService(apiUrl: String, apiClientToken: String)(implicit wsCli
     ) { resp =>
         resp.json.validate[GuestRegistrationResponse]
           .asEither
-          .bimap( _.mkString(","), response => UserIdWithGuestAccountToken.fromGuestRegistrationResponse(response))
+          .bimap(_.mkString(","), response => UserIdWithGuestAccountToken.fromGuestRegistrationResponse(response))
```

Comment:
IntelliJ did this against my will

Reply:
Haha this kept happening to me too!

Reply:
yep, it's enraging... hopefully this will put the issue to bed